signatureverifier/src/test/kotlin/io/github/rakutentech/signatureverifier/verification/AesEncryptorSpec.kt

```diff
@@ -0,0 +1,180 @@
+package io.github.rakutentech.signatureverifier.verification
+
+import com.nhaarman.mockitokotlin2.*
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import io.github.rakutentech.signatureverifier.RobolectricBaseSpec
+import io.github.rakutentech.signatureverifier.SignatureVerifier
+import org.amshove.kluent.*
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import java.lang.ClassCastException
+import java.lang.IllegalArgumentException
+import java.security.InvalidKeyException
+import java.security.KeyStore
+import java.security.NoSuchAlgorithmException
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+import javax.crypto.spec.GCMParameterSpec
+
+open class AesEncryptorSpec : RobolectricBaseSpec() {
+
+    internal val mockKeyStore = Mockito.mock(KeyStore::class.java)
+    private val mockKeyGenerator = Mockito.mock(AesKeyGenerator::class.java)
+    private val mockKeyEntry = Mockito.mock(KeyStore.SecretKeyEntry::class.java)
+
+    @Before
+    open fun setup() {
+        val key = generateAesKey()
+        SignatureVerifier.callback = null
+        When calling mockKeyGenerator.generateKey() itReturns key
+        When calling mockKeyEntry.secretKey itReturns key
+        When calling mockKeyStore.getEntry(any(), anyOrNull()) itReturns mockKeyEntry
+    }
+
+    @Test
+    fun `it should encrypt the data`() {
+        val encryptor = createAesEncryptor()
+
+        encryptor.encrypt("test data").shouldNotBeNull()
```

Comment:
I think it's better to operate on one 'subject' here to keep expectation results consistent. (I mean to have just one `encryptor.encrypt("test data")` result). The same goes for the test below

Reply:
updated
used variable to hold `data`